provdebug/Serializer.py

```diff
@@ -7,14 +7,14 @@
 class Serializer:
 
     @staticmethod
-    def save_records(records: List[DebugRecord]):
+    def save_records(records: List[DebugRecord], filename):
```

Comment:
Pass in the user's input filename as a parameter.

Reply:
Can you annotate it with `: str` just to be consistent with `records`?

Reply:
Yep!